src/analysis/import/import_osm.sh

```diff
@@ -145,6 +147,56 @@ osm2pgsql \
 # Delete downloaded temp OSM data
 rm -rf "${OSM_TEMPDIR}"
 
+# Create table for state residential speeds
+echo 'START: Importing State Default Speed Table'
+psql -h $NB_POSTGRESQL_HOST -U $NB_POSTGRESQL_USER -d $NB_POSTGRESQL_DB \
+     -c "CREATE TABLE IF NOT EXISTS \"state_speed\" (
+            state char(2),
+            fips_code_state smallint,
+            speed smallint
+        );"
+
+# Import state residential speeds file
+STATE_SPEED_FILENAME="state_fips_speed"
+STATE_SPEED_DOWNLOAD="/data/${STATE_SPEED_FILENAME}.csv"
+wget -nv -O "${STATE_SPEED_DOWNLOAD}" "https://s3.amazonaws.com/pfb-public-documents/${STATE_SPEED_FILENAME}.csv"
+psql -h $NB_POSTGRESQL_HOST -U $NB_POSTGRESQL_USER -d $NB_POSTGRESQL_DB \
+   -c "\copy state_speed FROM ${STATE_SPEED_DOWNLOAD} delimiter ',' csv header"
+
+# Set default residential speed for state
+STATE_DEFAULT=$( psql -h $NB_POSTGRESQL_HOST -U $NB_POSTGRESQL_USER -d $NB_POSTGRESQL_DB \
+      -t -c "SELECT state_speed.speed FROM state_speed WHERE state_speed.fips_code_state = ${PFB_STATE_FIPS}" )
+echo "DONE: Importing state default residential speed"
+
+# Create table for city residential speeds
+echo 'START: Importing City Default Speed Table'
+psql -h $NB_POSTGRESQL_HOST -U $NB_POSTGRESQL_USER -d $NB_POSTGRESQL_DB \
+     -c "CREATE TABLE IF NOT EXISTS \"city_speed\" (
+            city varchar,
+            state char(2),
+            fips_code_city integer,
```

Comment:
The FIPS code here should be stored as a `char` type. Like zip codes, they are identifiers with significant leading zeroes, rather than numbers we would do math with, and they are formed by concatenation.